apps/root/src/common/utils/transaction-history/index.ts

```diff
@@ -188,20 +188,21 @@ export const getTransactionValue = (tx: TransactionEvent, intl: ReturnType<typeo
       }`;
     case TransactionEventTypes.EARN_CREATED:
     case TransactionEventTypes.EARN_INCREASE:
-      return `${formatCurrencyAmount({ amount: tx.data.assetAmount.amount, token: tx.data.asset, intl })} ${
-        tx.data.asset.symbol
+      return `${formatCurrencyAmount({ amount: tx.data.depositAmount.amount, token: tx.data.depositToken, intl })} ${
+        tx.data.depositToken.symbol
       }`;
     case TransactionEventTypes.EARN_WITHDRAW:
-      const assetAmount = tx.data.withdrawn.find((withdrawn) => withdrawn.token.address === tx.data.assetAddress);
+      const withdrawnAsset = tx.data.withdrawn[0];
```

Comment:
why only the first one?

Reply:
It's a convention from the BE. They will always return all tokens, the asset first, and the amount = 0 will tell us which one's are withdrawn